src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Ccda/ConsultationNote/CDA_with_Embedded_PDF-expected.json

```diff
@@ -226,47 +223,6 @@
         "url": "Organization/c0da242a-0989-19d3-3bbd-75eb113418f2"
       }
     },
```

Comment:
Not generating practitioner any more?

Reply:
I add a condition that if one of the required properties is missing in sample data, then the resource will not be generated. In this case, the required property `name` of `practitioner` is missing in this sample data, so the `practitioner` isn't generated.
Do you think is it appropriate to add this condition or just generate the `practitioner ` resource anyway?